create_item_demo.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import dotenv
```

Comment:
These files are really useful for testing, but once they're working they tend to get abandoned and then end up quite messily in the codebase - I'm a big fan of deleting these as soon as you've got the key behaviour working in the app since that is now the source of truth for the right way to solve this problem!